README.md

```diff
@@ -1 +1,73 @@
-# equalizer
\ No newline at end of file
+# Wolox Equalizer
+[![circle-ci](https://img.shields.io/circleci/project/github/Wolox/wolox-equalizer.svg)](https://circleci.com/gh/Wolox/wolox-equalizer)
+[![npm](https://img.shields.io/npm/v/wolox-equalizer.svg)](https://www.npmjs.com/package/wolox-equalizer)
+
+[![FEArmy](./assets/FEA_icon.png)](https://github.com/orgs/Wolox/teams/front-end-army/members)
+## Features
+
+- Reset the default styles of the browser.
+- Contains our Wolox standards for normalizing front-end projects.
+
+## Installation
+
+#### npm
+```bash
+npm install --save wolox-equalizer
+```
+
+#### yarn
+```bash
+yarn add wolox-equalizer
+```
+
+## Usage
+
+1- Include the equalizer.scss to the root js file
+
+```js
+import 'wolox-equalizer/equalizer.scss';
+```
+
+
+## Inital color value
+
+1- All texts use **#717171** color by default.
+
+
+## Contributing
+
+1. Fork it
+2. Create your feature branch (`git checkout -b my-new-feature`)
+3. Commit your changes (`git commit -am 'Add some feature'`)
+4. Push to the branch (`git push origin my-new-feature`)
+5. Create new Pull Request
+
+## About
+
+This project is maintained by [Francisco Iglesias](https://github.com/frankiglesias) & [Braian Dickson](https://github.com/braiandickson) and it was written by [Wolox](http://www.wolox.com.ar).
+
+![Wolox](https://raw.githubusercontent.com/Wolox/press-kit/master/logos/logo_banner.png)
+
+## License
```

Comment:
Add a separate `LICENSE` file to suit Github standards

Reply:
same